schema/descriptor.schema.json

```diff
@@ -257,25 +296,50 @@
                         "id": "http://github.com/boutiques/boutiques-schema/input/max-list-entries",
                         "description": "Specify the maximum number of entries in the list. May only be used with List type inputs.",
                         "type": "number"
+                    },
+                    "absolute-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/absolute-path",
+                        "description": "Specifies that this input must be given as an absolute path. Only specifiable for File type inputs. Cannot be true if relative-path is true.",
+                        "type": "boolean"
+                    },
+                    "relative-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/relative-path",
```

Comment:
My interpretation is that if an input specifies `use-absolute-path=true`, then the platform must give that input an absolute-path value before it attempts to launch the container. The contract implied by this property is between the platform and the container. 

The contract between platform and user is another matter, and not covered by this property. The platform could require the user to give absolute paths to inputs with this property, or could translate user-provided relative paths to absolute. Whichever.